src/runtime/locale/en_gb.ts

```diff
@@ -0,0 +1,131 @@
+import type { Messages } from '../types'
+import { defineLocale } from '../composables/defineLocale'
+
+export default defineLocale<Messages>({
+  name: 'English (United Kingdom)',
+  code: 'en-GB',
+  messages: {
```

Comment:
@hywax I'd rather have this like you suggested!
```suggestion
  messages: en.messages
```